src/kms_config.js

```diff
@@ -3,18 +3,12 @@ const aws = require('aws-sdk');
 const load = (serverlessConfig = {}) => {
   const awsConfig = {};
 
-  const { service, variables } = serverlessConfig;
+  const { service } = serverlessConfig;
 
   if (service && service.provider && service.provider.region) {
     awsConfig.region = service.provider.region;
   }
 
-  const profile = variables.options['aws-profile'];
```

Comment:
This breaks on v3 because the serverless config object looks very different. It's not clear what the direct replacement would be, but this works locally so think it's fine. Will see what happens when I try this on CI 🙃